Detectors/MUON/MCH/Conditions/src/Plane.cxx

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "MCHConditions/Plane.h"
+
+#include "MCHConditions/Chamber.h"
+#include "MCHConditions/MeasurementType.h"
+#include "MCHConditions/Number.h"
+#include "MCHConditions/Side.h"
+#include <algorithm>
+#include <stdexcept>
+
+namespace o2::mch::dcs
+{
+
+Plane aliasToPlane(std::string_view dcsAlias)
+{
+  static constexpr auto chambersOfStation12 = {
+    Chamber::Ch00,
+    Chamber::Ch01,
+    Chamber::Ch02,
+    Chamber::Ch03};
+
+  auto chamber = aliasToChamber(dcsAlias);
+  if (std::find(begin(chambersOfStation12),
+                end(chambersOfStation12), chamber) != chambersOfStation12.end()) {
```

Comment:
you could use the function isQuadrant(...)

Reply:
indeed.